Project 3 Lol

```diff
@@ -0,0 +1,350 @@
+
+```
+For any question involving the use of Protege, please be sure to import Basic Formal Ontology (https://raw.githubusercontent.com/BFO-ontology/BFO/v2019-08-26/bfo_classes_only.owl) and the Relations Ontology (https://raw.githubusercontent.com/oborel/obo-relations/master/ro.owl)
+```
+
+1. In BFO and RO identify at least one object property for each of a-e that _should have the listed property, but which does not_; argue for your case, using examples. 
+``` 
+  (a)  Reflexive
+  - "has characteristic" should be a reflexive object property because everything has itself as a characteristic. 
+  -  I have myself as a characteristic: being myself distinguishes me from other people and things.
+  ...
+ - I was informed by Giacomo on the Project 3 Google Doc that this relation holds between a specifically dependent continuant and an independent continuant, which necessitates the two belong to different classes.
+ ...
+ - Many of the mereotopological relations in RO could be reflexive, but aren't.
+ - Ali and Jaron pointed out "overlaps" by explaining how an object is always perfectly overlapped with itself by virtue of the site it is contained in.
+ - I think their point tracks on to relations like "coincident with" and "colocalizes with".
+ - A process or entity is always perfectly coincident or colocalized with itself because of its identity.
+  
+  (b)  Transitive 
+ - "immersed in" should be a transitive object property.
+ - If object a is immersed in fluid b, and fluid b is immersed in fluid c, then object a is immersed in fluid c.
+ - I am also interpreting "immersed in" as one type of the "located in" relation, which is listed as a transitive object property.
+ 
+  (c)  Symmetric
+ - I am confused about the "before or simultaneous with" relation because I do not understand why those relationships are not kept separate.
+ - "simultaneous with" should definitely be a symmetric temporal relation.
+  
+  (d)  Functional 
+ - I think "has prototype" should be a functional object property.
+ - If a prototype is "some instance of a 'canonical'" entity, I am assuming there is only one prototype for every entity of interest.
+ 
+  (e)  Symmetric and Reflexive
+ - "overlaps" seems symmetric and reflexive to me (I got help from the project 3 Google doc on this one).
+  
+```
+
+2. In BFO and RO identify at least one object property for each of a-e that _should not have the listed property, but which does_; argue for your case, using examples. 
+```
+  (a)  Irreflexive
+ - I'm a little confused about the irreflexive property of the "has member" relation.
+ - The relation is specifically designed to exist between items and collections, so what about a collection with only one item in it?
+ - Can an object not be classed both as an 'item' and as a 'collection' of only itself?
+ 
```

Comment:
I don't think so. For example, you can write {Olivia} and Olivia. And the first one refers to the mathematical set containing the entity Olivia, but the second refers to you. They have very different properties - one is a person, the other a mathematical abstract object (a group). One is in space and time, the other isn't, etc.